cpp/src/decisiontree/decisiontree_impl.cuh

```diff
@@ -280,9 +283,27 @@ void DecisionTreeBase<T, L>::plant(
 
   total_temp_mem = tempmem->totalmem;
   MLCommon::TimerCPU timer;
-  grow_deep_tree(data, labels, rowids, n_sampled_rows, ncols,
-                 tree_params.max_features, dinfo.NLocalrows, sparsetree, treeid,
-                 tempmem);
+  if (tree_params.use_experimental_backend == true &&
+      tree_params.max_depth > 0 && tree_params.max_depth < 14 &&
+      tree_params.max_features == 1.0 &&
```

Comment:
Do we plan to support column sampling at some point?

Reply:
Yes.